noxfile.py

```diff
@@ -30,36 +32,31 @@ def session_check_requirements(session):
         session.run('gcp-devrel-py-tools', command, reqfile)
 
 
-def session_lint(session):
+@nox.session
+def lint(session):
     session.install('flake8', 'flake8-import-order')
     session.run(
         'flake8', '--exclude=env,.nox,._config.py,.tox',
         '--import-order-style=google', '.')
 
 
 def run_test(session, dir, toxargs):
-    shutil.copy('config.py', dir)
     session.chdir(dir)
     session.run('tox', *(toxargs or []))
 
 
+@nox.session
 @nox.parametrize('dir', DIRS)
-def session_run_tests(session, dir=None, toxargs=None):
+def run_tests(session, dir=None, toxargs=None):
     """Run all tests for all directories (slow!)"""
     run_test(session, dir, toxargs)
 
 
-def session_run_one_test(session):
```

Comment:
I don't _think_ this is needed anymore, since there is syntax for running [a single version of a parametrized session](https://nox.thea.codes/en/stable/usage.html#specifying-parametrized-sessions ):